### PR TITLE
[Android] Compile fixes: don't link with -lrt, no DMLC_LOG_STACK_TRAC…

### PR DESCRIPTION
**Pull request by larroy**
…E, fopen64 -> std::fopen

### CODE REVIEW

**Comment by larroy** [comment]
@KellenSunderland

**Comment by larroy** [comment]
@piiswrong

File: include/dmlc/base.h [hunk 1]
```diff
@@ -37,12 +37,16 @@
 #define DMLC_LOG_CUSTOMIZE 0
 #endif
 
+#ifdef __ANDROID__
+    #define fopen64 std::fopen
```
Comment (piiswrong) [inline include/dmlc/base.h]: I thought we had this defined somewhere

**Comment by larroy** [comment]
Ok, merged it down below as you suggested.

**Comment by larroy** [comment]
https://github.com/dmlc/mshadow/pull/280
https://github.com/apache/incubator-mxnet/pull/7416

**Pull request merged**